src/box/tuple.h

```diff
@@ -1021,7 +1025,8 @@ tuple_field_i64(struct tuple *tuple, uint32_t fieldno, int64_t *out)
 	default:
 		diag_set(ClientError, ER_FIELD_TYPE,
 			 int2str(fieldno + TUPLE_INDEX_BASE),
-			 field_type_strs[FIELD_TYPE_INTEGER]);
+			 field_type_strs[FIELD_TYPE_INTEGER],
+			 field_type_strs[actual_type]);
```

Comment:
It should be mp_type_strs instead of field_type_strs: actual_type is type of value (mp), no field type

Reply:
Fix